### PR TITLE
Bump vtk-js

### PR DESCRIPTION
**Pull request by boucaud**
This includes a small fix necessary for upgrade, because DataAccessHelpers now need to be explicitly registered, or imported.

### CODE REVIEW

File: src/io/GlanceVtkJsReader.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import DataAccessHelper from 'vtk.js/Sources/IO/Core/DataAccessHelper';
+import vtkJSZipDataAccessHelper from 'vtk.js/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper';
```
Comment (jourdain) [inline src/io/GlanceVtkJsReader.js:1]: You just need to import it so the `.get('zip')` will work. No need to do anything more.

Comment (jourdain) [reply src/io/GlanceVtkJsReader.js:1]: Just add `import 'vtk.js/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper';` at the top of the file

File: src/io/GlanceVtkJsReader.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line no-unused-vars
+import vtkJSZipDataAccessHelper from 'vtk.js/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper';
```
Comment (jourdain) [inline src/io/GlanceVtkJsReader.js:2]: `import 'vtk.js/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper';`

Comment (jourdain) [reply src/io/GlanceVtkJsReader.js:2]: That way you don't need that comment for eslint

Comment (boucaud) [reply src/io/GlanceVtkJsReader.js]: Thanks !

**Comment by floryst** [comment]
Can you push the package-lock.json changes? It should be a fairly small change from what I can tell.

**Review by floryst — CHANGES_REQUESTED** [review]
Requesting changes so I know not to merge. Having package-lock.json committed helps in pinning a package version for CI.

**Comment by boucaud** [comment]
> Can you push the package-lock.json changes? It should be a fairly small change from what I can tell.

done

**Comment by floryst** [comment]
Thanks!

**Pull request merged**